typings/index.d.ts

```diff
@@ -1032,6 +1032,7 @@ export class GuildMember extends PartialTextBasedChannel(Base) {
   public readonly displayHexColor: HexColorString;
   public readonly displayName: string;
   public guild: Guild;
+  public readonly communicationDisabledUntil: Date | null;
```

Comment:
Missing communicationDisabledUntilTimestamp, method and #edit parameter

Reply:
added